packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -233,12 +238,30 @@ export class Pane extends Foundation<PaneHandledProps, PaneUnhandledProps, PaneS
         return (
             <button
                 className={this.props.managedClasses.pane_resizeHandle}
-                aria-hidden={true}
                 onMouseDown={this.onMouseDown}
+                onKeyDown={this.onKeyDown}
+                aria-hidden={true}
             />
         );
     }
 
+    /**
+     * Handle keyPress
+     */
+    public onKeyDown = (e: React.KeyboardEvent<HTMLButtonElement>): void => {
+        switch (e.keyCode) {
+            case KeyCodes.arrowLeft /* tslint:disable-next-line */:
```

Comment:
Can we add a shift key check and increment by 10 when shift is held?

Reply:
@nicholasrice - added and updated. Thx